src/v1/bookSearch/RecentSearch.tsx

```diff
@@ -0,0 +1,41 @@
+import type { APIBookRecentSearchResponse } from '@/types/book';
+
+import Button from '@/v1/base/Button';
+
+type RecentSearchProps = {
+  recentSearches: APIBookRecentSearchResponse[] | undefined;
+  setInputSearchValue: (searchValue: string) => void;
+};
+
+const RecentSearch = ({
+  recentSearches,
+  setInputSearchValue,
+}: RecentSearchProps) => {
+  return (
+    <section className="flex flex-col gap-[1.7rem]">
+      <h2 className="h-[2.4rem] text-lg">최근 검색어</h2>
+      {recentSearches ? (
+        <div className="relative flex h-[4.5rem] w-[calc(100%+2rem)] gap-[1.5rem] overflow-x-scroll whitespace-nowrap pb-[1rem]">
+          {recentSearches.map(value => (
+            <Button
+              key={`${value.keyword}-${value.modifiedAt}`}
+              size="medium"
+              fill={true}
+              fullRadius={true}
+              colorScheme="main-light"
+              onClick={() => setInputSearchValue(value.keyword)}
+            >
+              {value.keyword}
+            </Button>
+          ))}
+        </div>
```

Comment:
p1;

최근 검색어 영역이 디자인과 달리 크게 구현된 것 같아요. `Button` 컴포넌트의 크기와 버튼 사이의 간격을 줄이면 좋을 것 같아요!

```suggestion
        <div className="relative flex h-[4.5rem] w-[calc(100%+2rem)] gap-[1rem] overflow-x-scroll whitespace-nowrap pb-[1rem]">
          {recentSearches.map(value => (
            <Button
              key={`${value.keyword}-${value.modifiedAt}`}
              size="small"
              fill={true}
              fullRadius={true}
              colorScheme="main-light"
              onClick={() => setInputSearchValue(value.keyword)}
            >
              {value.keyword}
            </Button>
          ))}
        </div>
```